packages/gasket-fetch/node.js

```diff
@@ -0,0 +1,2 @@
+const fetch = require('node-fetch');
```

Comment:
Won't this fail if consumers don't install `devDependencies`?

Reply:
Yes, I didn't pay attention on the install. 🤦‍♂️